README.md

```diff
@@ -1 +1,111 @@
-# portfolio-project
\ No newline at end of file
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+
+# 📖 [Portfolio: setup and mobile first]
+
+**[Portfolio: setup and mobile first]** is a simple project that show a page of a portfolio that discribe the work, experience and contacts of the portfolio owner.
+
+## 🛠 Built With
+
+### Tech Stack
+
+Tech Stack
+HTML
+CSS
+JS
+Git/GitHub
+node.js
+Boostrap 5
+
+### Key Features
+
+- **[A webpage that gives you all the information of the owner]**
+
+## 🚀 Live Demo
+
+- [Comming Soon]
+
+## 💻 Getting Started
+
+When you click on the page, the browser shows you the information of the owner.
+
+### Prerequisites
+
+Clone this repository to your desired folder and open html file.
+
+### Setup
+
+Clone this repository to your desired folder:
+Example commands:
+
+```sh
+  cd portfolio-project
+  git clone https://github.com/ntimanu/portfolio-project
+```
+
+### Install
+
+Install this project with:
+
+<!--
+Example command:
+```sh
+  cd Capstone-1-Project
+  npm install
+```
+--->
+
+### Usage
+
+To run the project, execute the following command:
+
+<!--
+Example command:
+```sh
+  open live server
+```
+--->
+
+## 👥 Authors
+
+👤 **Emmanuel Ntihinyuka**
+
+- GitHub: [@ntimanu](https://github.com/ntimanu)
+- Twitter:[@emmantihinyuka](https://twitter.com/emmantihinyuka)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/ntihinyuka-emmanuel-511890104/)
+
+## 🔭 Future Features
+
+- [ ] **[Add more forms]**
+- [ ] **[Add more pages with information]**
+- [ ] **[Add some Javascript]**
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+## ⭐️ Show your support
+
+If you like this project feel free to share with your friends.
+
+## 🙏 Acknowledgments
+
+I would like to thank Microverse for this amazing exercise.
+
+## 📝 License
+
+This project is [MIT](https://choosealicense.com/licenses/mit/) licensed.
```

Comment:
- [ ] Good job at updating the readme with your information, but I see here that when I try to click on the link of the MIT LICENCE it is redirecting me to the unspecific page for the LICENCE, so kindly try to put the correct link for the MIT LICENCE for the project 

> Check below to see the difference 😊⤵️
> Click on the image to open it at a new tab in the browser if you can not see it clearly 😊

| My LICENCE Page| Your LICENCE Page| 
| --- | --- |
|    ![image](https://user-images.githubusercontent.com/83360636/219104404-1b7d0313-97e5-48fb-8567-fa846713ee0d.png)  |    ![image](https://user-images.githubusercontent.com/83360636/219104310-960826ce-cf05-4929-90ef-ad3c330dde35.png)   |

Reply:
Done

Reply:
Done